### DISCUSSION TITLE
[multisite] Allow to filter by group site and all sites

### DISCUSSION THREAD

**Discussion by eduardoalonsoalbella** [craftcms/cms#9932] [Ideas]
Hi, In "entries" menu, I would like to be able to filter by group site and also be able to filter by all sites, currently in craft cms 3.7.14 we can filter only by 1 site.

See origin of this discussion: https://github.com/craftcms/cms/discussions/9926

I think this feature would improve entries filtering capabilities in craft cms, allowing user to be able to see al entries in an specific site group or listing all entries.

[All]
[Group Site 1]
- Site 1.1
- Site 1.2
- Site 1....
- Site 1.n

[Group Site 2]
- Site 2.1
- Site 2.2
- Site 2....
- Site 2.n

**Comment by brandonkelly** [comment]
The `_elements/sitemenu` template is used in a few different places. Which specific “entries menu” are you referring to?

**Reply by eduardoalonsoalbella** [reply]
/cms/entries/pages

![Captura de pantalla 2021-10-06 a las 15 28 21-fullpage](https://user-images.githubusercontent.com/90317987/136211909-8f0c8206-c4ca-4d26-b1e0-9d8fd8300b0a.png)

**Comment by eduardoalonsoalbella** [comment]
@brandonkelly any thoughts?

**Reply by brandonkelly** [reply]
It’s on my radar but not something we would get around to before 4.0.
